configs/mender_grub_config

```diff
@@ -13,7 +17,7 @@ MENDER_GRUB_EFI_INTEGRATION=y
 MENDER_GRUB_KERNEL_BOOT_ARGS=""
 
 # grub-mender-grubenv is the Mender integration for the GRUB bootloader
-MENDER_GRUBENV_VERSION="1.3.0"
+MENDER_GRUBENV_VERSION="6ffc280e64dd7b6bbc52f7747609f11fbdd1a057"
```

Comment:
Same here, I think this checksum is outdated.

Reply:
Okey... it is already fixed in https://github.com/mendersoftware/mender-convert/pull/203/ :)